src/index.js

```diff
@@ -65,6 +64,12 @@ export default class Loader extends EventEmitter {
       })
   }
 
+  _parseDataUri (data, mimeType, isBase64) {
+    const headers = new Headers({ 'Content-Type': mimeType })
```

Comment:
Because of the `fetch` spec, `Headers` is now a standard class on the window. I wanted to stick to that rather than introduce Node APIs into the browser.